corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -228,9 +231,10 @@ hqDefine("data_dictionary/js/data_dictionary", [
 
         self.newCaseProperty = function () {
             if (_.isString(self.newPropertyName())) {
-                var prop = propertyListItem(self.newPropertyName(), false, '', self.activeCaseType(), '', '', {});
+                var prop = propertyListItem(self.newPropertyName(), '', false, '', self.activeCaseType(), '', '', {});
```

Comment:
should this one be `self.newPropertyName()` instead of blank value since by default we are keeping them same or we are not doing that anymore?

Reply:
thanks for pointing that out. b3ec55464c452eba288843ee58943191d5b57ff4